Jenkinsfile

```
@@ -1 +1,5 @@
-buildPlugin(useAci: false, configurations: buildPlugin.recommendedConfigurations())
+buildPlugin(useAci: false, configurations: [
```

```suggestion
buildPlugin(configurations: [
```

as this is the default

---

I want this to be explicit in case the default changes in the future since the version of Git installed on the Windows ACI agents currently breaks this plugin's tests.